draft-ietf-httpbis-digest-headers.md

```diff
@@ -236,7 +236,8 @@ transformations (eg. transfer codings for HTTP/1.1 see 6.1 of
 {{resource-representation}} contains several examples to help illustrate those effects.
 
 A representation digest consists of
-the value of a checksum computed on the entire selected `representation data` of a resource
+the value of a checksum computed on the entire selected `representation data`
+(see Section 7 of {{SEMANTICS}}) of a resource identified according to Section 7.3.2 of {{SEMANTICS}}
 together with an indication of the algorithm used (and any parameters)
```

Comment:
This new text look technically fine but the sentence is a bit long winded now, especially with the
parameter comment in brackets. How about some editorializing like this, which intentially avoids saying anything about parameters.

```
A representation digest consists of a `digest-algorithm` followed by an `encoded digest output`.

~~~ abnf
   representation-data-digest = digest-algorithm "="
                                <encoded digest output>
~~~

The `digest-algorithm` identifies both an algorithm and output encoding format which are applied
to a representation. A checksum value is first computed on the entire selected `representation data`
(see Section 7 of {{SEMANTICS}}) of a resource identified according to Section 7.3.2 of {{SEMANTICS}}.
This is then encoded using the required format.

The list of defined `digest-algorithms` is provided in {{algorithms}}.

The example below shows the `sha-256` digest-algorithm which uses base64 encoding.

~~~ example
   sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
~~~
```

Reply:
If this is technically ok, I suggest to:

- merge and close the issue;
- resolve #850 (parameters);
- rewording in another PR.

In the rewording, we could even avoid mentioning the "encoding" part as I think it should only be mentioned in the `digest-algorithm` section.

WDYT?

Reply:
sgtm